server/util/strategicLogger.ts

```diff
@@ -43,28 +43,26 @@ function getAmazonAliasByGithub(pg, alias) {
 async function addWhitelistedContrib(pg, contrib, projId) {
   const desc = contrib.title;
   const date = contrib.created_at;
-  const alias = (await getAmazonAliasByGithub(pg, contrib.user.login))
+  const alias = (await getCompanyAliasByGithub(pg, contrib.user.login))
```

Comment:
No need for the additional parens here.

Reply:
This is needed if you look at line 47, I'm accessing .company_alias from what the promise returns, so I have to await the promise and then call it:
const alias = (await getCompanyAliasByGithub(pg, contrib,user.login)).company_alias

This linter pushed the .company_alias to the next line.